src/commands/create_command.py

```diff
@@ -1,32 +1,24 @@
-import json, os
+import os
 from commands.abstract_command import *
-from commands.keys import Keys
-from output.exceptions import *
+from model.database import *
 
 
 class CreateCommand(AbstractCommand):
-    def __init__(self, schema_file):
-        self.validate(schema_file, Keys.SCHEMA_PATH)
-        schema_file = os.path.join(Keys.SCHEMA_PATH, schema_file)
-        with open(schema_file, 'r') as file:
-            self.data = json.load(file)
-        self.check_database()
-        self.path = os.path.join(Keys.DATABASE_PATH, self.data[Keys.DATABASE])
+    def __init__(self, schema_name):
+        self.validate(schema_name, Keys.SCHEMA_PATH)
+        self.schema_name = schema_name
 
     def execute(self):
-        os.makedirs(self.path, exist_ok=True)
-        for table in self.data[Keys.TABLES]:
-            t_path = os.path.join(self.path, table[Keys.NAME])
-            os.makedirs(t_path, exist_ok=True)
+        self.__create_database__()
```

Comment:
You are calling `__create_database` only here and this is the only method is called here. so why don't move the code from `__create_database` to here :)